vnext/ReactUWP/Views/TextInputViewManager.cpp

```diff
@@ -280,6 +281,28 @@ void TextInputShadowNode::registerEvents() {
         }
       });
 
+  m_controlKeyDownRevoker = control.KeyDown(
+      winrt::auto_revoke, [=](auto &&, winrt::KeyRoutedEventArgs const &args) {
+        if (args.Key() == winrt::VirtualKey::Enter && !args.Handled()) {
+          auto instance = wkinstance.lock();
```

Comment:
>auto instance = wkinstance.lock(); [](start = 10, length = 34)

this will be more readable:
if (auto instance = wkinstance.lock()) {
}

Reply:
Will do

---
In reply to: [335118934](https://github.com/microsoft/react-native-windows/pull/3423#discussion_r335118934) [](ancestors = 335118934)